dali/python/nvidia/dali/plugin/jax/iterator.py

```diff
@@ -173,6 +173,7 @@ def __init__(
             prepare_first_batch=prepare_first_batch)
 
         self._first_batch = None
+        self._peek = None
```

Comment:
Do we need `_peek ` as member in the regular iterator?

Reply:
Removed, leftover from some experiments.